cypress/e2e/gui/engage-sphere.spec.cy.js

```diff
@@ -0,0 +1,378 @@
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
+describe('Customers', () => {
+    beforeEach(() => {
+        cy.clearCookies();
+        cy.setCookie('cookieConsent', 'accepted');
+        cy.visit('/');
+        cy.contains('h2', 'Hi there!', { timeout: 5000 })
```

Comment:
```suggestion
        cy.contains('h2', 'Hi there!')
```

Crei já ter perguntado sobre o motivo de ter sobrescrito este _timeout_, não?